crates/mun_codegen/src/lib.rs

```diff
@@ -1,11 +1,20 @@
+/// This library generates machine code from HIR using inkwell which is a safe wrapper around LLVM.
+
 #[macro_use]
 extern crate lazy_static;
```

Comment:
`extern crate` is no longer necessary for macros (since 2018). You can just `use crate::macro_name;`